### PR TITLE
CRYP-344: Transaction list is not formatted correctly when filter / sort is applied

### PR DESCRIPTION
**Pull request by alexbara2000**
Fixed the UI bug where the badges were not formatted right.
https://cryptify.atlassian.net/browse/CRYP-344 
image

### CODE REVIEW

**Review by jason-gerard — APPROVED** [review]
Add a screenshot with many filters applied

**Comment by alexbara2000** [comment]
image

File: packages/client/screens/TransactionsListScreen.tsx [hunk 1]
```diff
@@ -166,59 +167,62 @@ export default function TransactionsListScreen(props: HomeStackScreenProps<"Tran
...
+            </VStack>
         </View>
     );
 }
 
 const styles = StyleSheet.create({
     view: {
-        flex: 1,
         paddingTop: 10,
+        Color: "#FFFFFF",
```
Comment (domsec) [inline packages/client/screens/TransactionsListScreen.tsx:224]: Lowercase c for color. Do we even need this though? Isn't the screen/view automatically white?

**Comment by alexbara2000** [comment]
> image

It is scrollable and it still works as intended

**Review by domsec — APPROVED** [review]

**Pull request merged**